packages/dev/loaders/src/OBJ/solidParser.ts

```diff
@@ -537,6 +551,28 @@ export class SolidParser {
         return line.startsWith("g");
     }
 
+    private static _GetZbrushMRGB(line: string, notParse: boolean) {
+        if (!line.startsWith("mrgb")) return null;
+        line = line.replace("mrgb", "").trim();
+        // if include vertex color , not load mrgb anymore
+        if (notParse) return [];
+        const regex = /[a-z0-9]/g;
+        const regArray = line.match(regex);
+        if (!regArray || regArray.length % 8 !== 0) {
+            return null;
+        }
+        const array = [];

```

Comment:
This is an `any` array. Can we make this either `{r: number, g: number, b: number}[]` or just directly construct Babylon `Color4` objects here (e.g. `Color4[]`)?